src/core/vlite.css

```diff
@@ -60,7 +60,7 @@
 		.v-controlBar,
 		.v-bigPlay {
 			position: fixed;
-			z-index: 2147483647;
+			z-index: 7;
```

Comment:
Increase the value to keep a little margin I think (min 10, 15 perhaps)

Reply:
The latest `z-index` is 7 (https://github.com/vlitejs/vlite/pull/155#issuecomment-2675841019)
I update the `z-index` to 8, we should increase it on occurence.